transifex_catalogs/fa.json

```diff
@@ -1,50 +1,102 @@
+  "brand_guide_14": "برای پکیج های طراحی، راهنما و مستندات، مخزن کد گیت هاب (github/dcrdesign) را مشاهده فرمایید.",
+  "brand_guide_2": "بسته لوگو",
+  "brand_guide_3": "دانلود بسته لوگو",
+  "brand_guide_4": "رنگ های اصلی",
+  "brand_guide_5": "آبی اصلی",
+  "brand_guide_6": "فیروزه ای",
+  "brand_guide_7": "آبی تیره",
+  "brand_guide_8": "تایپوگرافی",
+  "brand_guide_9": "رنگ‌های تکمیلی",
+  "brand_guide_title": "جعبه ابزار استفاده سریع",
+  "community_community_title": "پلتفرم های انجمن",
+  "community_publications_title": "انتشارات",
+  "community_support_title": "پشتیبانی",
+  "community_support_warning": "ادمین ها هرگز برای شما DM ارسال نمی کنند یا از طریق DM پشتیبانی نمی کنند. هیچ اطلاعات شخصی مانند رمز کیف پول خود را به کسی که به شما کمک می کند ارائه ندهید.",
+  "description": "دیکْرِد، یک ارز دیجیتال خودمختار است. این ارز با یک سیستم اجماع ترکیبی(هیبریدی) ساخته شده است تا غیرمتمرکز، پایدار و دارای قوانین خود ساخته ای باشد که ذینفعان (سهامداران) آن را تعیین می کنند.",
+  "downloads_app_coinomi": "بیت‌کوین، اتیریم، و بیش از 500 دارایی زنجیره بلوکی دیگر را ایمن ذخیره، مدیریت، و تبدیل کنید.",
+  "downloads_app_commandline": "نصب کننده/آپدیت خودکار و چند پلتفورمی برای برنامه های خط فرمان.",
+  "downloads_app_dcrdex": "کیف پول سبک به همراه پشتیبانی از DEX و چند ارزی. فاقد استیکینگ یا قابلیت های کیف پول پیشرفته.",
+  "downloads_app_decrediton": "کیف پول رابط کاربری گرافیکی برای Windows، macOS و Linux.",
+  "downloads_app_exodus": "Exodus\n برنامه ای کاربرپسند است که قابلیت های ایمن سازی، مدیریت و تبادل دارایی های زنجیره بلوکی را در یک جا جمع می کند.",
+  "downloads_app_guarda": "Guarda\n با کیف پول‌های چند ارزی مبتنی بر وب، رایانه، و دستگاه همراه ارائه می‌شود.",
+  "downloads_app_ledger": "Ledger Live\n کیف پولی مبتنی بر وب است که با کیف پول سخت افزاری Ledger ایمن‌سازی شده است.",
+  "downloads_app_mobilewallets": "کیف پول های تلفن همراه دیکْرِد(Decred) موجود برای پلتفرم های اندروید و iOS",
+  "downloads_app_trezor": "Trezor\n یک کیف پول سخت افزاری است که توسط Decrediton and Exodus wallets پشتیبانی میشود.",
+  "downloads_app_trust": "Trust Wallet\nیک کیف پول چند ارزی و ایمن برای اندروید و iOS است.",
+  "downloads_category_core_software": "نرم‌افزار اصلی",
+  "downloads_category_hardware_wallets": "کیف پول های سخت افزاری",
+  "downloads_category_third_party_wallets": "کیف پول های شخص ثالث",
+  "downloads_description": "اگر تازه با دیکْرِد(Decred) آشنا شده‌اید و می‌خواهید اطلاعات عمیق‌تری داشته باشید، حتماً <a href='https://docs.decred.org' target='_blank' rel='noopener noreferrer'> مستندات </a> را بخوانید. برای افراد محتاط در زمینه امنیت لطفا <a href='https://docs.decred.org/advanced/verifying-binaries/' target='_blank' rel='noopener noreferrer'>these این مراحل </a> را برای تایید کد های باینری اپلیکیشن های دیکردتان و کیت خط فرمان دنبال کنید.",

```

Comment:
Looks like the word `these` was missed?

Reply:
"These" was translated but still not removed. Removed it.